idr0037-study.txt

```diff
@@ -14,15 +14,15 @@ Study External URL	http://www.hipsci.org/
 Study Public Release Date							
 							
 # Study Publication							
-Study PubMed ID							
-Study Publication Title							
-Study Author List							
-Study PMC ID							
-Study DOI							
+Study PubMed ID		30784590					
+Study Publication Title	Identifying Extrinsic versus Intrinsic Drivers of Variation in Cell Behavior in Human iPSC Lines from Healthy Donors
+Study Author List	Alessandra Vigilante, Anna Laddach, Nathalie Moens, Ruta Meleckyte, Andreas Leha, Arsham Ghahramani, Oliver J. Culley, Annie Kathuria, Chloe Hurling, Alice Vickers, Erika Wiseman, Mukul Tewary, Peter W. Zandstra, HipSci Consortium, Richard Durbin, Franca Fraternali, Oliver Stegle, Ewan Birney, Nicholas M. Luscombe, Davide Danovi, Fiona M. Watt
+Study PMC ID	https://www.cell.com/cell-reports/fulltext/S2211-1247(19)30131-7
```

Comment:
From https://www.ncbi.nlm.nih.gov/pubmed/30784590 I don't see this a PMC ID yet. This is the published URL